parachain/src/chain_spec.rs

```diff
@@ -513,6 +514,8 @@ fn mainnet_genesis(
         vault_registry: VaultRegistryConfig {
             minimum_collateral_vault: vec![(CurrencyId::KSM, 0)],
             punishment_delay: DAYS,
+            system_collateral_ceiling: vec![(CurrencyId::KSM, 5533_000_000_000_000)], /* 5533 ksm, about 2 mm USD at
```

Comment:
The `CurrencyId` type has the number of decimals, can you figure out some way to derive this number from that? i.e. `5533 * CurrencyId::KSM::decimals()`